fabtests/ubertest/config.c

```diff
@@ -717,6 +717,11 @@ int fts_info_is_valid(void)
 	if (test_info.test_type == FT_TEST_UNIT &&
 	    test_info.ep_type == FI_EP_DGRAM)
 		return 0;
+	if ((test_info.test_type == FT_TEST_UNIT) &&
+	    (opts.options & FT_OPT_ENABLE_HMEM)) {
+		if (test_info.test_class == (FT_CAP_ATOMIC))
```

Comment:
These two 'if' statements can be merged.

Reply:
Fixed this as well in the latest change.